portal-frontend/src/app/features/public/search/public-search.component.ts

```diff
@@ -128,7 +128,8 @@ export class PublicSearchComponent implements OnInit, OnDestroy {
     private codeService: CodeService,
     private statusService: StatusService,
     private toastService: ToastService,
-    private titleService: Title
+    private titleService: Title,
+    private elementRef: ElementRef,
```

Comment:
It's not really clear what element this references. Might be better to use a `ViewChild`

Reply:
i have kept it consistent with what we use in decision page